src/ralph/admin/sitetrees.py

```diff
@@ -39,63 +39,75 @@
                 'data_center.change_virtualserver'
             ],
             perms_mode_all=False,
+            access_loggedin=True,
```

Comment:
what do you think about creating some `ralph_item` which will be partial to item with `access_loggedin=True` set?